pvdeg/weather.py

```diff
@@ -82,19 +82,28 @@ def get(database, id=None, geospatial=False, **kwargs):
             raise NameError("Weather database not found.")
 
         if "relative_humidity" not in weather_df.columns:
-            print('Column "relative_humidity" not found in DataFrame. Calculating...')
+            print('Column "relative_humidity" not found in DataFrame. Calculating...' , end='')
             weather_df = humidity._ambient(weather_df)
+            print('\r                                                                        ', end='')
```

Comment:
What it is doing here is that when it prints that message that it is calculating the relative humidity, it prints the message without a line feed and when it is done with the calculation it does a carriage return without a line feed and writes over the message so that one doesn't see the annoying message on their screen except for when the calculations are in progress. If you have a better way to do this, please do so.